src/components/timeline/Row.svelte

```diff
@@ -340,7 +345,7 @@
         </g>
       </svg>
       <!-- Loading indicator -->
-      {#if hasResourceLayer && ($fetchingResources || $fetchingResourcesExternal)}
+      {#if hasResourceLayer && ($fetchingResources || $fetchingResourcesExternal || $fetchingResourcesExternalNames)}
```

Comment:
Pretty sure we don't actually have to wait for `fetchingResourcesExternalNames` here since the layer doesn't rely on them? Might be useful to show a loading indicator in the timeline layer editor resource picker but that might not be worth it right now since we're refactoring that whole thing in the near future. But if you feel inspired to do it go for it.

Reply:
Yeah I guess that's true, I just added it here since it should always finish before `fetchingResourcesExternal`. I'll just remove the flag for now and we can use it after the refactor if we need.